.github/actions/Ubuntu18.04-build/entrypoint.sh

```diff
@@ -11,6 +11,7 @@ git clone --depth 1 -b $1 https://github.com/ImageMagick/ImageMagick.git
 
 cd ImageMagick && \
     mv /tmp/debian . && \
+    sed -i "1s/7.1.0-16/$1/g" "debian/changelog" "debian/README.Debian" && \
```

Comment:
I would prefer that those two files contains a placeholder so became more explicit that is meant to be replaced and you don't risk to replace unintended strings